tests/integration/test_dss.py

```diff
@@ -45,23 +46,63 @@ def test_initialize_creates_dss(cleanup_after_initialize) -> None:
 
     # Check if the dss notebook pvc exists using kubectl
     kubectl_result = subprocess.run(
-        ["kubectl", "get", "pvc", "notebooks", "-n", "dss"], capture_output=True, text=True
+        ["kubectl", "get", "pvc", "notebooks", "-n", DSS_NAMESPACE], capture_output=True, text=True
     )
     assert "notebooks" in kubectl_result.stdout
 
 
-@pytest.fixture()
-def cleanup_after_initialize():
-    """Cleans up resources that might have been deployed by dss initialize.
+def test_create_notebook(cleanup_after_tests) -> None:
+    """
+    Tests that `dss create-notebook` successfully creates a notebook as expected.
+
+    Must be run after `dss initialize`
+    """
+    # TODO: is there a better way to initialize this?  Maybe an optional argument to the test?
+    kubeconfig_file = "~/.kube/config"
+    kubeconfig = lightkube.KubeConfig.from_file(kubeconfig_file)
+    lightkube_client = lightkube.Client(kubeconfig)
+
+    notebook_name = "test-notebook"
+    notebook_image = "kubeflownotebookswg/jupyter-scipy:v1.8.0"
+
+    result = subprocess.run(
+        [
+            DSS_NAMESPACE,
+            "create-notebook",
+            "--name",
+            notebook_name,
+            "--image",
+            notebook_image,
+            "--kubeconfig",
+            kubeconfig_file,
+        ],
+        capture_output=True,
+        text=True,
+        timeout=60 * 4,
+    )
+
+    # Check if the command executed successfully
+    print(result.stdout)
```

Comment:
I believe `prints` are lost in integration checks. Should we log it instead?

Reply:
actually, both `print` and `log` are filtered out by pytest by default.  But specifying `pytest -s` (which is done in `tox.ini`) tells pytest to make the output of both visible.

We can still log though instead of print.  I don't know which is better for tests

Reply:
I see. It's a minor thing and I will leave it up to you, just ensure the logs are actually printed (if you have to modify the tox, let's do it).